src/pushex.ts

```diff
@@ -31,7 +31,6 @@ export class Pushex {
       socketReconnectAlgorithm: config.socketReconnectAlgorithm || DEFAULT_SOCKET_RECONNECT_ALGORITHM,
     }
 
-    this.subscriptions = {}
```

Comment:
Moved this to line 18.

Reply:
I'm undoing this particular line because it's not part of my build tooling currently. `npm run build` fails